react-native/react/native/remote-component.desktop.js

```diff
@@ -34,8 +34,9 @@ export default class RemoteComponent extends Component {
 
     const componentRequireName = this.props.component
     const substore = this.props.substore
+    const hot = process.env.HOT === 'true'
 
-    this.remoteWindow.loadUrl(`file://${__dirname}/remoteComponent.html#${componentRequireName || ''}:${substore || ''}`)
+    this.remoteWindow.loadUrl(`file://${path.resolve('../react-native/react/native/remoteComponent.html')}?component=${componentRequireName || ''}&substore=${substore || ''}&hot=${!!hot}`)
```

Comment:
I do to turn it into a bool

On Thursday, December 10, 2015, Marco Munizaga <notifications@github.com>
wrote:

> In react-native/react/native/remote-component.desktop.js
> <https://github.com/keybase/client/pull/1507#discussion_r47304289>:
>
> >
> > -    this.remoteWindow.loadUrl(`file://${__dirname}/remoteComponent.html#${componentRequireName || ''}:${substore || ''}`)
> > +    this.remoteWindow.loadUrl(`file://${path.resolve('../react-native/react/native/remoteComponent.html')}?component=${componentRequireName || ''}&substore=${substore || ''}&hot=${!!hot}`)
>
> [image: :sheep:] You don't really need to do !!hot
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1507/files#r47304289>.
>